manifests/init.pp

```diff
@@ -17,6 +19,20 @@
     purge   => $config_dir_purge,
   }
 
+  if str2bool($cron_purge) {
```

Comment:
`True` as a bareword is considered a String in the Puppet DSL.  The boolean values are `true` and `false`.